### ISSUE TITLE
Request for support solidity 0.5.0 release 

### ISSUE THREAD

**Issue by noyonthe1** [pouladzade/Seriality#1]
Could you please update this for solidity 0.5.0 release? 

I have faced a lot of compilation error. I fixed few which don't involve with assembly. But for assembly like this below I think its better to ask for you help here. 

```
/Users/hasnaeenferdous/startbahn/dev/github/artcontract/app/contracts/src/BytesToTypes.sol:59:13: SyntaxError: The use of labels is disallowed. Please use "if", "switch", "for" or function calls instead.
            loop:
            ^--^
,/Users/hasnaeenferdous/startbahn/dev/github/artcontract/app/contracts/src/BytesToTypes.sol:59:13: SyntaxError: Jump instructions and labels are low-level EVM features that can lead to incorrect stack access. Because of that they are discouraged. Please consider using "switch", "if" or "for" statements instead.
            loop:
            ^--^
,/Users/hasnaeenferdous/startbahn/dev/github/artcontract/app/contracts/src/BytesToTypes.sol:64:13: SyntaxError: Jump instructions and labels are low-level EVM features that can lead to incorrect stack access. Because of that they are discouraged. Please consider using "switch", "if" or "for" statements instead.
            jumpi(loop , lt(loop_index , chunk_count))
            ^----------------------------------------^
,/Users/hasnaeenferdous/startbahn/dev/github/artcontract/app/contracts/src/TypesToBytes.sol:45:9: SyntaxError: The use of labels is disallowed. Please use "if", "switch", "for" or function calls instead.
        loop:
        ^--^
,/Users/hasnaeenferdous/startbahn/dev/github/artcontract/app/contracts/src/TypesToBytes.sol:45:9: SyntaxError: Jump instructions and labels are low-level EVM features that can lead to incorrect stack access. Because of that they are discouraged. Please consider using "switch", "if" or "for" statements instead.
        loop:
        ^--^
,/Users/hasnaeenferdous/startbahn/dev/github/artcontract/app/contracts/src/TypesToBytes.sol:50:13: SyntaxError: Jump instructions and labels are low-level EVM features that can lead to incorrect stack access. Because of that they are discouraged. Please consider using "switch", "if" or "for" statements instead.
            jumpi(loop , lt(index,stack_size))
```

**Comment by pouladzade** [comment]
Thank you so much for opening  the first issue.
Sure, I will upgrade the code to support the latest version of Solidity probably by first of next week.

**Comment by noyonthe1** [comment]
> Thank you so much for opening the first issue.
> Sure, I will upgrade the code to support the latest version of Solidity probably by first of next week.

Actually, I'm working on something where I need this library in 0.5.0 very much. :) thanks for your quick heads up and support. Please let me know how can I help to make it faster, I can push my already created changes into a different branch, only if you want?

**Comment by pouladzade** [comment]
@noyonthe1 I have already updated the code to fix this issue and work with the Solidity 0.5.0.
Would you please help me by getting the latest code and see if the issues already have been fixed?

**Comment by noyonthe1** [comment]
Thanks, @pouladzade for upgrading this. Build error gone for me on 0.5.0.
